src/App.js

```diff
@@ -1,14 +1,39 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import Overlay from "./Components/Overlay";
 import SearchBar from "./Components/SearchBar";
-
+import axios from "axios";
 import "./App.css";
+
 function App() {
-  const [search, setSearch] = useState(false);
+  const [searchTerm, setSearchTerm] = useState("");
+  const [searchResult, setSearchResults] = useState([]);
+  const fetchWordSearchResults = async () => {
```

Comment:
I would recommend adding a re-usable utility function to do web requests. The function should take in the request type, endpoint URL, and params, and should return the request result. Also, move the exception handling into this function.